sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -122,9 +122,13 @@ try {
 
 Send a single event/message using [azure-messaging-eventhubs][azure-messaging-eventhubs] with tracing enabled.
 
-Users can additionally pass the value of the current tracing span to the EventData object with key **PARENT_TRACE_CONTEXT_KEY** on the [Context][context] object:
+Users can additionally pass custom value of the trace context to the EventData object with key **PARENT_TRACE_CONTEXT_KEY** on the [Context][context] object.
+
+Please refer to [EventHubs samples](https://github.com/Azure/azure-sdk-for-java/blob/10a18ccc2f20cad6004ae90d64f22009d65e9ef7/sdk/eventhubs/azure-messaging-eventhubs/src/samples/java/com/azure/messaging/eventhubs/PublishEventsTracingWithCustomContextSample.java)
```

Comment:
Will we have to update this to main in the future?

Reply:
I'd try to use the link style of the ones below. Also, it is "Event Hubs".

Reply:
> Will we have to update this to main in the future?

yes, unfortunately, I need to provide a working link first and it doesn't exist in main yet.